README.md

```diff
@@ -0,0 +1,25 @@
+# PostgreSQL
```

Comment:
You should rebase. Tom merged a commit https://github.com/postgres/postgres/commit/e2b73f4a4de6c2df6a1e623ad06b42f1bb2471ad which makes some of the same changes as you.

I'd also include the links which he has in his PR (eg. add a link to docs somewhere).

Reply:
"Installation from Source Code" docs link was added.